tools/linter/components/SupersededOperationsFile.ts

```diff
@@ -0,0 +1,23 @@
+import FileValidator from './base/FileValidator'
+import ajv from 'ajv'
+import fs from 'fs'
+import YAML from 'yaml'
+import { type ValidationError } from '../../types'
+
+export default class SupersededOperationsFile extends FileValidator {
+  JSON_SCHEMA_PATH = '../json_schemas/_superseded_operations.yaml'
+
+  validate (): ValidationError[] {
+    return [
+      this.validate_json_schema()
+    ].filter(e => e) as ValidationError[]
+  }
+
+  validate_json_schema (): ValidationError | undefined {
+    const schema = YAML.parse(fs.readFileSync(this.JSON_SCHEMA_PATH, 'utf8'))
+    const validator = (new ajv()).compile(schema)
+    if (!validator(this.spec())) {
+      return this.error(`File content does not match JSON schema found in '${this.JSON_SCHEMA_PATH}':\n ${JSON.stringify(validator.errors, null, 2)}`)
+    }
+  }
+}
```

Comment:
This feels like a pretty generic validator, consider extracting it to a SchemaFileValidator (matching the name of FileValidator), and renaming to SupersededOperationsFileValidator (also matching the name of FileValidator).

Reply:
This validator will also handle sorting of the HTTP verbs as well.

I will add JSON schemas for `_info` and `_global_parameters` files and will DRY this logic when creating validators for them.